src/control/controlindicatortimer.cpp

```diff
@@ -0,0 +1,32 @@
+#include "control/controlindicatortimer.h"
+
+#include "control/controlobject.h"
+#include "moc_controlindicatortimer.cpp"
+
+namespace mixxx {
+
+ControlIndicatorTimer::ControlIndicatorTimer(QObject* pParent)
+        : QObject(pParent),
+          m_pCOIndicator250millis(std::make_unique<ControlObject>(
+                  ConfigKey("[Master]", "indicator_250millis"))),
+          m_pCOIndicator500millis(std::make_unique<ControlObject>(
+                  ConfigKey("[Master]", "indicator_500millis"))),
+          m_toggleIndicator500millisOnNextTimeout(true) {
+    m_pCOIndicator250millis->setReadOnly();
+    m_pCOIndicator500millis->setReadOnly();
+    connect(&m_timer, &QTimer::timeout, this, &ControlIndicatorTimer::slotTimeout);
+    m_timer.start(250);
+}
+
+void ControlIndicatorTimer::slotTimeout() {
```

Comment:
I suggest to use a lambda for this slot and get rid of the member variable.

Reply:
Can you elaborate? I don't see how to avoid the member variable and use a lambda here. Except if I use two timers, which is undesired because then there is no guarantee that both indicators are in sync.

Reply:
Doesn't work. In C++ we can't capture mutable from the environment like in Rust.